c2corg_ui/static/js/card.js

```diff
@@ -164,13 +244,64 @@ app.CardController.prototype.createURL = function() {
   var loc = window.location.pathname;
   // Don't create links on edit and add pages.
   if (loc.indexOf('/edit/') === -1 && loc.indexOf('/add') === -1) {
-    return this.url_.buildDocumentUrl(
-      this.type, this.doc['document_id'], this.doc['locales'][0]);
+    if (this.type == 'feeds') {
+      return this.url_.buildDocumentUrl(
+        app.utils.getDoctype(this.doc['document']['type']), this.doc['document']['document_id'], this.doc['document']['locales'][0]);
+    } else {
+      return this.url_.buildDocumentUrl(
+        this.type, this.doc['document_id'], this.doc['locales'][0]);
+    }
   }
 };
 
 
 /**
+ * @param {string} filename
+ * @param {string} suffix
+ * @return {string}
+ * @export
+ */
+app.CardController.prototype.createImageUrl = function(filename, suffix) {
+  return this.imageUrl_ + app.utils.createImageUrl(filename, suffix);
+};
+
+
+/**
+ * @param {Array} areas
+ * @return {string | undefined}
+ * @export
+ */
+app.CardController.prototype.createURLArea = function(areas) {
+  if (areas !== undefined) {
+    if (areas.length > 0) {
```

Comment:
Instead of the 2 lines perhaps you could write
``if (areas && areas.length) {``?

Reply:
fixed